.github/workflows/release.yaml

```diff
@@ -32,6 +37,14 @@ jobs:
       - name: Run bin/create-release
         run: 'GH_TOKEN=${{ secrets.GITHUB_TOKEN }} bin/create-release ${{ github.event.inputs.commit_sha }} ${{ github.event.inputs.release_number }}'
 
+      - name: Tag cloud-deploy-infra
```

Comment:
(blocking) Add the `if success` check so this doesn't run if creating the release fails. Should be like doing the failure [here](https://github.com/civiform/civiform/blob/45d55b4e80bb4fbbd2059587856693994d65d064/.github/workflows/tests.yaml#L202).